sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -322,7 +322,26 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder {
   }
 
   /**
-   * Create a [[CreateTable]] logical plan.
+   * Create a table, returning a [[CreateTable]] logical plan.
+   *
+   * This is used to create data source tables, which is handled through
```

Comment:
nit: I know this is copied from `visitCreateTable`'s comment. So the sentence "which is ..." looks weird and redundant here.